presidio-anonymizer/presidio_anonymizer/engine/anonymizers/fpe.py

```diff
@@ -0,0 +1,17 @@
+# TODO implement + test
+from anonymizers.anonymizer import AnonymizerAbstract
+
+
+class FPE(AnonymizerAbstract):
```

Comment:
What is FPE?
Can the name be more verbose?

Reply:
Please view previous discussion :)